src/silx/gui/widgets/OverlayMixIn.py

```diff
@@ -12,9 +12,28 @@ class OverlayMixIn:
     .. warning:: Any class inheriting from this mixin must also inherit from a QWidget.
     """
 
-    def __init__(self, parent):
+    def __init__(
+        self,
+        parent,
+        alignment: qt.Qt.AlignmentFlag = qt.Qt.AlignCenter,
+        alignment_offsets: tuple[int] = (0, 0),
```

Comment:
To be consistent with Qt, it's best to have setters for features rather than passing too many arguments in the `__init__`, so best to remove `alignment` and `alignment_offsets` arguments from here and provide `get|setAlignment` and `get|setAlignmentOffsets`